### ISSUE TITLE
dragging file or folder to terminal doesn't quote or escape correctly

### ISSUE THREAD

**Issue by brad-provident** [microsoft/vscode#146192]
Issue Type: <b>Bug</b>

In the past, I have been able to drag a file or folder from the Explorer panel to the terminal, and the path would be entered there with either quotes or escaping the spaces (I don't remember which), so that it could be used in, e.g. a "cd" command. This no longer works. The path is neither escaped nor put into quotes, so it doesn't work correctly if there are spaces in a directory or file in that path.

VS Code version: Code 1.65.2 (c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1, 2022-03-10T14:33:49.188Z)
OS version: Darwin x64 21.4.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz (16 x 2400)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|32.00GB (0.20GB free)|
|Process Argv|--crash-reporter-id 0fb5047f-c1a3-4cb9-8f0d-24f030cb7c3f|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (50)</summary>

Extension|Author (truncated)|Version
---|---|---
csso|100|1.9.0
terminal-commands|adr|1.0.5
vscode-icalendar|af4|1.0.1
vscode-caniuse|aga|0.5.0
vscode-caniuse|aka|0.5.4
vbscript-language-server-client|and|1.1.0
icons-carbon|ant|0.2.4
filewatcher|App|1.0.3
speech|bie|0.1.0
task-panel|Bit|0.2.0
path-intellisense|chr|2.8.0
gitlens|eam|12.0.5
vscode-html-css|ecm|1.11.0
json-pretty-printer|eus|1.1.0
cshtml|fir|0.1.3
git-extensions-for-vs-code|for|1.1.3
editorenhancements|gur|1.0.1
openchrome|hua|0.0.3
githd|hui|2.2.4
vscode-edit-csv|jan|0.6.8
vscode-browser-sync|jas|1.3.1
wovo|jim|1.1.1
svg|joc|1.4.17
classic-asp-html|jtj|0.1.2
vbscript|kar|0.5.5
l13-diff|L13|1.0.3
VBScript|lug|0.0.2
markdown-shortcuts|mdi|0.12.0
ecdc|mit|1.4.0
HTMLHint|mka|0.10.0
prettify-json|moh|0.0.3
csharp|ms-|1.24.1
team|ms-|1.161.1
vscode-css-peek|pra|4.2.0
vscode-xml|red|0.19.1
live-sass|rit|3.0.0
LiveServer|rit|5.7.5
color-manager|roy|0.6.3
partial-diff|ryu|1.4.3
code-settings-sync|Sha|3.4.3
svg-snippets|sid|1.0.1
uridecode|sry|0.3.5
open-in-browser|tec|2.0.0
vscode-icons-mac|way|7.25.3
html-snippets|wge|0.1.5
firefox-debug|yur|0.1.0
markdown-all-in-one|yzh|3.4.0
json|Zai|2.0.2
html-css-class-completion|Zig|1.20.0
z-tf-utils|zol|0.1.2

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30443607
pythontb:30283811
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263cf:30335440
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vsclayoutctrt:30451275
dsvsc009:30452663
pythonvsnew555:30457759
vscscmwlcmt:30438805
vscgsvid1:30447480
vscaac:30438847
pynewfile477:30450038

```

</details>

**Comment by meganrogge** [comment]
it's working for me in insider's. 
I believe this is already fixed, can you see if you can reproduce in the [Insiders build](https://code.visualstudio.com/insiders)?
/"needs-more-info"

**Comment by brad-provident** [comment]
This is fixed in the latest version now (1.66.0).
